.pytool/Plugin/ImageValidation/ReadMe.md

```diff
@@ -282,16 +257,18 @@ This setting is only used if the alignment requirements specify multiple require
 "ALIGNMENT_LOGIC_SEP" : "<Logical Operator>"
 ```
 
+#### IMAGE_BASE
+
+This setting is used to specify what the base address of an image should be
+
+```json
+"IMAGE_BASE": <integer>
```

Comment:
Can this be done per binary, not just per binary class? Or would the advice be to add to the ignore list? Say if one driver had an image_base req? I think the ignore list is okay, since that would be a rare case.

Reply:
My opinion is to just add it to the ignore list. If this is a feature requested, we could probably update the IGNORE_LIST config section to specify only ignoring certain tests, rather than all tests as is now. In my opinion, not worth the effort to add the functionality quite yet.